ufv.js

```diff
@@ -273,19 +278,62 @@ UFV.prototype.handleStreamRequest = function(request) {
             fps = expectedFPS;
           }
 
-          bitrate = videoInfo["max_bit_rate"];
+          vbitrate = videoInfo["max_bit_rate"];
+        }
+
+        let audioInfo = request["audio"];
+        if (audioInfo) {
+          abitrate = audioInfo["max_bit_rate"];
+          asamplerate = audioInfo["sample_rate"];
         }
 
         let targetAddress = sessionInfo["address"];
         let targetVideoPort = sessionInfo["video_port"];
         let videoKey = sessionInfo["video_srtp"];
         let videoSsrc = sessionInfo["video_ssrc"];
-
-        let ffmpegCommand = this.ffmpegSource + ' -threads 0 -vcodec '+vcodec+' -an -pix_fmt yuv420p -r '+
-        fps +' -f rawvideo -tune zerolatency -vf scale='+ width +':'+ height +' -b:v '+ bitrate +'k -bufsize '+
-         bitrate +'k -payload_type 99 -ssrc '+ videoSsrc +' -f rtp -srtp_out_suite AES_CM_128_HMAC_SHA1_80 -srtp_out_params '+
-         videoKey.toString('base64')+' srtp://'+targetAddress+':'+targetVideoPort+'?rtcpport='+targetVideoPort+
-         '&localrtcpport='+targetVideoPort+'&pkt_size=1378';
+        let targetAudioPort = sessionInfo["audio_port"];
+        let audioKey = sessionInfo["audio_srtp"];
+        let audioSsrc = sessionInfo["audio_ssrc"];
+
+        let ffmpegCommand = [
+          ...this.ffmpegSource.split(' '),
+          '-threads', '0',
+          '-map', '0:v:0',
+          '-vcodec', vcodec,
+          '-pix_fmt', 'yuv420p',
+          '-r', fps,
+          '-f', 'rawvideo',
+          '-tune', 'zerolatency',
+          '-vf', 'scale='+ width +':'+ height,
+          '-b:v', vbitrate +'k',
+          '-bufsize', vbitrate +'k',
+          '-payload_type', '99',
+          '-ssrc', videoSsrc,
+          '-f', 'rtp',
+          '-srtp_out_suite', 'AES_CM_128_HMAC_SHA1_80',
+          '-srtp_out_params', videoKey.toString('base64'),
+          `srtp://${targetAddress}:${targetVideoPort}?rtcpport=${targetVideoPort}&localrtcpport=${targetVideoPort}&pkt_size=1378`,
+        ];
+
+        if(this.audio) {
+          ffmpegCommand.push(
+            '-map', '0:a:0',
+            '-acodec', acodec,
+            '-profile:a', 'aac_eld',
+            '-flags', 'global_header',
+            '-f', 'null',
+            '-ar', asamplerate + 'k',
+            '-b:a', abitrate + 'k',
+            '-bufsize', abitrate + 'k',
+            '-ac', '1',
+            '-payload_type', '110',
+            '-ssrc', audioSsrc,
+            '-f', 'rtp',
+            '-srtp_out_suite', 'AES_CM_128_HMAC_SHA1_80',
+            '-srtp_out_params', audioKey.toString('base64'),
+            `srtp://${targetAddress}:${targetAudioPort}?rtcpport=${targetAudioPort}&localrtcpport=${targetAudioPort}&pkt_size=1378`,
+          );
+        }
         console.log(ffmpegCommand);
         let ffmpeg = spawn('ffmpeg', ffmpegCommand.split(' '), {env: process.env});
```

Comment:
`ffmpegCommand.split` should just be `ffmpegCommand` since it is now an array.

Reply:
Whoops, must've missed that when committing… Should be fixed now!